taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+              <ul>
+                <li>
+                  <a href="#figure4">Figure 4. macOS “New Bookmark” setup screen</a>
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="#references">References</a>
+        </li>
+      </ul>
+
+      <h3>Data Transfer and Management Guide</h3>
+
+      <p>
+        Transferring Portal data from your local machine to
+        <a href="#storage">one of TACC's remote storage systems</a> can be accomplished using two methods: command line tools (<code>scp</code>, <code>sftp</code>,
+        <code>rsync</code>) and graphical tools (Cyberduck).
+      </p>
+
+      <h4 id="table1">Data Transfer Methods</h4>
+      <table style="max-width: 500px">
+        <tbody>
+          <tr>
+            <th>Usage Mode</th>
+            <th>Transfer Method</th>
+          </tr>
+          <tr>
+            <td>Command Line Tool</td>
+            <td>
+              <a target="_blank" href="https://man7.org/linux/man-pages/man1/scp.1.html"><code>scp</code></a>
+            </td>
+          </tr>
+          <tr>
+            <td>Command Line Tool</td>
+            <td>
+              <a target="_blank" href="https://man7.org/linux/man-pages/man1/sftp.1.html"><code>sftp</code></a>
+            </td>
+          </tr>
+          <tr>
+            <td>Command Line Tool</td>
+            <td>
+              <a target="_blank" href="https://man7.org/linux/man-pages/man1/rsync.1.html"><code>rsync</code></a>
+            </td>
+          </tr>
+          <tr>
+            <td>Graphical Tool</td>
+            <td>
+              <a target="_blank" href="https://cyberduck.io/">Cyberduck</a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h4 id="storage">What is TACC's Storage Server?</h4>
+
+      <p>
+        A TACC Storage System is one of TACC’s systems dedicated to storage and data management. It can be used for storing protected data on a secure location for this Portal. As an Portal user, you have access to a secure server for this Portal, like “
+        <code><samp>sub.domain</samp>.tacc.utexas.edu</code>
+        ” (which this document will refer to as <kbd>host</kbd>), as well as the secure folder for the Portal, like “
+        <code>/<kbd>secure-server-root</kbd>/projects/<kbd>PORTAL</kbd></code>
+        ” (which this document will refer to as <kbd>/transfer/directory/path</kbd>). This secure system is to be used exclusively for transferring and accessing Portal data.
+      </p>
+      <h4 id="rsync_prereqs">Prerequisites for Portal User</h4>
+
+      <p>There are two requirements needed prior to transferring data:</p>
+
+      <ul>
+        <li>A TACC Account</li>
+        <li>
+          Multi-Factor Authentication (MFA) pairing with the
+          <a target="_blank" href="https://portal.tacc.utexas.edu/tutorials/multifactor-authentication#tacctokenapp">TACC Token app</a>
+        </li>
+      </ul>
+
+      <p>
```

Comment:
All portal users will need to have created a TACC account in the TACC User Portal which can be accessed at `https"//portal.tacc.utexas.edu`. 

If you have forgotten your TACC account credentials, please refer to your email for a message titled "TACC Account Request Confirmation" or go to `https://portal.tacc.utexas.edu/password-reset/-/password/forgot-username` for further assistance in account recovery.

Access to all TACC resources requires a completed Multi-Factor Authentication pairing with your TACC credentials. To set up MFA, please reference the tutorial at `https://portal.tacc.utexas.edu/tutorials/multifactor-authentication`.

Reply:
Done (in a soon to be pushed commit). I further edited these paragraphs, though:

> […] or use the TACC Portal's [password reset form](https://portal.tacc.utexas.edu/password-reset/-/password/) or [username recovery form](https://portal.tacc.utexas.edu/password-reset/-/password/forgot-username).

> […] To set up MFA, please reference TACC Portal's [Multi-Factor Authentication tutorial](https://portal.tacc.utexas.edu/tutorials/multifactor-authentication).